### DISCUSSION TITLE
Cannot commit anymore

### DISCUSSION THREAD

**Discussion by yngwi** [NeogitOrg/neogit#377] [Q&A]
Hi,
I can't exactly tell since when, but I can't commit any more. When pressing "c + c" in the Neogit status window, after the first "c" the commit popup opens, but after the second "c" the popup closes immediately with the following error message: 

![image](https://user-images.githubusercontent.com/963440/197391857-65b063cb-2ab9-487b-bc48-9a0fb720b299.png)

Can you tell me what this is referring to? How can I fix it? I'm using signed commits, git and (for instance) lazygit works without problem.

I'm grateful for any help!
Best,
Daniel

**Comment by ttytm** [comment]
Hey @yngwi  

Are you still experience this with the current version?

There was a lot going on with the repo. Probably nothing you could do until a fix was pushed upstream. Alternatively, locking on an older commit it worked with.

**Comment by yngwi** [comment]
Hi, yes, I just tried on windows and got the following error in `messages` after trying to open the commit window with `c c`:

```
hint: Waiting for your editor to close the file...
C:\Program Files\Neovim\bin\nvim.exe --headless --clean --noplugin -n -R -c "set runtimepath^=C:\Users\user\.vim\plug
ged\neogit" -c "lua require(""neogit.client"").client()": line 1: C:Program: command not found
error: There was a problem with the editor 'C:\Program Files\Neovim\bin\nvim.exe --headless --clean --noplugin -n -R -
c "set runtimepath^=C:\Users\djell\.vim\plugged\neogit" -c "lua require(""neogit.client"").client()"'.
Please supply the message using either -m or -F option.
```
